src/sa_web/static/js/routes.js

```diff
@@ -7,6 +7,11 @@ var Shareabouts = Shareabouts || {};
     routes: {
       '': 'viewMap',
       'place/new': 'newPlace',
+      'place/new/observation': 'newPlaceObservation',
+      'place/new/question': 'newPlaceQuestion',
+      'place/new/idea': 'newPlaceIdea',
+      'place/new/complaint': 'newPlaceComplaint',
+      'place/new/greenwall': 'newPlaceGreenwall',
```

Comment:
I think the best way is to extend this view, and add it into our flavor specific code under `src/flavors/duwamish_flavor/static/js/view/routes.js` where we add only the changes that we need. That new view will then override this one. I can implement this and push it up as an example.